### ISSUE TITLE
Quick swap between search providers 

### ISSUE THREAD

**Issue by Mrfiregem** [TobiasBielefeld/Simple-Search#14]
Coming off of the app SearchBar EX, the only large feature I think is missing is the ability to temporarily change providers easily through a pullout or dropdown rather than opening settings to do so.

**Comment by LeSnake04** [comment]
I miss this Feture too

**Comment by trymeouteh** [comment]
Searchbar Ex is a great app but it is closed source and not on F-Driod which means LineageOS users cannot use Searchbar Ex. Simple Search does the trick but this features would be handy indeed.

**Comment by paul1149** [comment]
Definitely. This is a natural enhancement and would be awesome.

**Comment by LeSnake04** [comment]
I think this app is discontinued...

**Comment by wikd** [comment]
Saw a update so I thought I'd mention again, all this app needs is some quick way to swap between search engines and it would be perfect. Maybe just as simple as "g search term" for Google etc.

**Comment by KeronCyst** [comment]
Agreed. Is the dev reading this at all? Having a change-search-engine button directly on the main page where you type the query is essential. It would literally make this app a game-changer.

**Comment by TobiasBielefeld** [comment]
This feature has been added in Version v2.2. There is also a dialog in the settings to select which search engines to show in the drop down list
